Gemfile

```diff
@@ -8,6 +8,7 @@ gemspec
 
 group :development, :test do
   gem 'bundler'
+  gem 'irb'
```

Comment:
Adding this to the Gemfile fixes the following problem: when I hit the up arrow to go back through history, the cursor starts at the begging of the line. I want the cursor to be at the end of the line when I am arrowing up/down through history. Including `irb` in the Gemfile causes that to happen, somehow, which it otherwise doesn't.